Hi3Helper.Core/Lang/en_US.json

```diff
@@ -1574,6 +1574,7 @@
     "Graphics_EffectsQ": "Effects Quality",
     "Graphics_ShadingQ": "Shading Quality",
     "Graphics_Distortion": "Distortion",
+    "Graphics_AnisotropicSampling": "Anisotropic Sampling",

```

Comment:
Technically this is already available under _GenshinGameSettingsPage, I think we can reuse it instead of telling localizer to resubmit exactly the same thing